#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-add-shebangs-and-fix-permissions.dpatch by  <wolfi@sigxcpu.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to fix permissions and add missing shebang-lines in scripts.
## DP: Taken from upstream's Bug tracker at
## DP: https://bugs.launchpad.net/inkscape/+bug/602005
## DP: original patch by Leo Jackson
## DP: 2011-05-30: Modified by Alex Valavanis <valavanisalex@gmail.com>:
## DP: * Do not add shebangs to JavaScript files.  They cannot be run as
## DP:   standalone scripts.
## DP: * Install JavaScript files using _DATA rule rather than _SCRIPTS
## DP:   rule to avoid making them executable.

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Base.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Base.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Base.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Base.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code128.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code128.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code128.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code128.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python 
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code39.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code39.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code39.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code39.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code39Ext.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code39Ext.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code39Ext.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code39Ext.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code93.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code93.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/Code93.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/Code93.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/EAN13.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/EAN13.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/EAN13.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/EAN13.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/EAN5.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/EAN5.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/EAN5.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/EAN5.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python 
 ''' 
 Copyright (C) 2007 Martin Owens 
 Copyright (C) 2009 Aaron C Spike 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/EAN8.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/EAN8.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/EAN8.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/EAN8.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/RM4CC.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/RM4CC.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/RM4CC.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/RM4CC.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/UPCA.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/UPCA.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/UPCA.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/UPCA.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/UPCE.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/UPCE.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/UPCE.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/UPCE.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/__init__.py inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/__init__.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Barcode/__init__.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Barcode/__init__.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,4 +1,4 @@
-
+#!/usr/bin/env python
 '''
 Barcodes SVG Extention
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/Makefile.am inkscape_0.48.1-2ubuntu5/share/extensions/Makefile.am
--- inkscape_0.48.1-2ubuntu5~/share/extensions/Makefile.am	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/Makefile.am	2011-05-30 17:46:40.853386475 +0100
@@ -73,13 +73,9 @@
 	inkex.py \
 	Inkscape.pm \
 	inkwebeffect.py \
-	inkweb.js \
 	interp.py \
 	interp_att_g.py \
-	jessyInk.js \
 	jessyInk_autoTexts.py \
-	jessyInk_core_mouseHandler_noclick.js \
-	jessyInk_core_mouseHandler_zoomControl.js \
 	jessyInk_effects.py \
 	jessyInk_export.py \
 	jessyInk_install.py \
@@ -161,6 +157,10 @@
 	yocto_css.py
 
 otherstuff = \
+	inkweb.js \
+	jessyInk.js \
+	jessyInk_core_mouseHandler_noclick.js \
+	jessyInk_core_mouseHandler_zoomControl.js \
 	aisvg.xslt \
 	colors.xml \
 	jessyInk_video.svg \
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/SpSVG.pm inkscape_0.48.1-2ubuntu5/share/extensions/SpSVG.pm
--- inkscape_0.48.1-2ubuntu5~/share/extensions/SpSVG.pm	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/SpSVG.pm	2011-05-30 17:43:57.893386425 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 #
 # SpSVG
 # 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_blackandwhite.py inkscape_0.48.1-2ubuntu5/share/extensions/color_blackandwhite.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_blackandwhite.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_blackandwhite.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect,sys
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_brighter.py inkscape_0.48.1-2ubuntu5/share/extensions/color_brighter.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_brighter.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_brighter.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_custom.py inkscape_0.48.1-2ubuntu5/share/extensions/color_custom.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_custom.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_custom.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -22,4 +23,4 @@
     return '%02x%02x%02x' % (int(round(r2*255)),int(round(g2*255)),int(round(b2*255)))
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_darker.py inkscape_0.48.1-2ubuntu5/share/extensions/color_darker.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_darker.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_darker.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_desaturate.py inkscape_0.48.1-2ubuntu5/share/extensions/color_desaturate.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_desaturate.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_desaturate.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -7,4 +8,4 @@
     return '%02x%02x%02x' % (ig,ig,ig)
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_grayscale.py inkscape_0.48.1-2ubuntu5/share/extensions/color_grayscale.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_grayscale.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_grayscale.py	2011-05-30 17:43:57.893386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -11,4 +12,4 @@
     return '%02x%02x%02x' % (ig,ig,ig)
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_lesshue.py inkscape_0.48.1-2ubuntu5/share/extensions/color_lesshue.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_lesshue.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_lesshue.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect, inkex
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_lesslight.py inkscape_0.48.1-2ubuntu5/share/extensions/color_lesslight.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_lesslight.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_lesslight.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect, inkex
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_lesssaturation.py inkscape_0.48.1-2ubuntu5/share/extensions/color_lesssaturation.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_lesssaturation.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_lesssaturation.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect, inkex
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_morehue.py inkscape_0.48.1-2ubuntu5/share/extensions/color_morehue.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_morehue.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_morehue.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect, inkex
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_morelight.py inkscape_0.48.1-2ubuntu5/share/extensions/color_morelight.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_morelight.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_morelight.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect, inkex
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_moresaturation.py inkscape_0.48.1-2ubuntu5/share/extensions/color_moresaturation.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_moresaturation.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_moresaturation.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect, inkex
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_negative.py inkscape_0.48.1-2ubuntu5/share/extensions/color_negative.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_negative.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_negative.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -5,4 +6,4 @@
     return '%02x%02x%02x' % (255-r,255-g,255-b)
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_randomize.py inkscape_0.48.1-2ubuntu5/share/extensions/color_randomize.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_randomize.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_randomize.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect,random,inkex
 
 class C(coloreffect.ColorEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_removeblue.py inkscape_0.48.1-2ubuntu5/share/extensions/color_removeblue.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_removeblue.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_removeblue.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -5,4 +6,4 @@
     return '%02x%02x%02x' % (r,g,0)
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_removegreen.py inkscape_0.48.1-2ubuntu5/share/extensions/color_removegreen.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_removegreen.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_removegreen.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -5,4 +6,4 @@
     return '%02x%02x%02x' % (r,0,b)
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_removered.py inkscape_0.48.1-2ubuntu5/share/extensions/color_removered.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_removered.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_removered.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -5,4 +6,4 @@
     return '%02x%02x%02x' % (0,g,b)
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_replace.py inkscape_0.48.1-2ubuntu5/share/extensions/color_replace.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_replace.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_replace.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 import inkex
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/color_rgbbarrel.py inkscape_0.48.1-2ubuntu5/share/extensions/color_rgbbarrel.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/color_rgbbarrel.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/color_rgbbarrel.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import coloreffect
 
 class C(coloreffect.ColorEffect):
@@ -5,4 +6,4 @@
     return '%02x%02x%02x' % (b,r,g)
 
 c = C()
-c.affect()
\ No newline at end of file
+c.affect()
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/dxf_templates.py inkscape_0.48.1-2ubuntu5/share/extensions/dxf_templates.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/dxf_templates.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/dxf_templates.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 r14_header = '''  0
 SECTION
   2
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/render_alphabetsoup_config.py inkscape_0.48.1-2ubuntu5/share/extensions/render_alphabetsoup_config.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/render_alphabetsoup_config.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/render_alphabetsoup_config.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 # Syntax format: (raise your hand if you know lisp :-)
 #
 # 'state0':  ("file.svg", ( ( ('state1', dx, dy, T-B, L|R),),
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/render_barcode.py inkscape_0.48.1-2ubuntu5/share/extensions/render_barcode.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/render_barcode.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/render_barcode.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 '''
 Copyright (C) 2007 Martin Owens
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/run_command.py inkscape_0.48.1-2ubuntu5/share/extensions/run_command.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/run_command.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/run_command.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import os
 import sys
 import tempfile
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/svg_and_media_zip_output.py inkscape_0.48.1-2ubuntu5/share/extensions/svg_and_media_zip_output.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/svg_and_media_zip_output.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/svg_and_media_zip_output.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/env python
 """
 svg_and_media_zip_output.py
 An extention which collects all images to the documents directory and
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/svg_regex.py inkscape_0.48.1-2ubuntu5/share/extensions/svg_regex.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/svg_regex.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/svg_regex.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 # This software is OSI Certified Open Source Software.
 # OSI Certified is a certification mark of the Open Source Initiative.
 # 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_braille.py inkscape_0.48.1-2ubuntu5/share/extensions/text_braille.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_braille.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_braille.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 #encoding: utf-8
 import chardataeffect, inkex, string
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_flipcase.py inkscape_0.48.1-2ubuntu5/share/extensions/text_flipcase.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_flipcase.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_flipcase.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import chardataeffect, inkex, string
 
 class C(chardataeffect.CharDataEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_lowercase.py inkscape_0.48.1-2ubuntu5/share/extensions/text_lowercase.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_lowercase.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_lowercase.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import chardataeffect, inkex, string
 
 class C(chardataeffect.CharDataEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_randomcase.py inkscape_0.48.1-2ubuntu5/share/extensions/text_randomcase.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_randomcase.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_randomcase.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import chardataeffect, inkex, string
 
 import random
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_replace.py inkscape_0.48.1-2ubuntu5/share/extensions/text_replace.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_replace.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_replace.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import chardataeffect, inkex, string
 
 class C(chardataeffect.CharDataEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_sentencecase.py inkscape_0.48.1-2ubuntu5/share/extensions/text_sentencecase.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_sentencecase.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_sentencecase.py	2011-05-30 17:43:57.903386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import chardataeffect, inkex, string
 
 class C(chardataeffect.CharDataEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_titlecase.py inkscape_0.48.1-2ubuntu5/share/extensions/text_titlecase.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_titlecase.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_titlecase.py	2011-05-30 17:43:57.913386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import chardataeffect, inkex, string
 
 class C(chardataeffect.CharDataEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/text_uppercase.py inkscape_0.48.1-2ubuntu5/share/extensions/text_uppercase.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/text_uppercase.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/text_uppercase.py	2011-05-30 17:43:57.913386425 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 import chardataeffect, inkex, string
 
 class C(chardataeffect.CharDataEffect):
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape_0.48.1-2ubuntu5~/share/extensions/voronoi.py inkscape_0.48.1-2ubuntu5/share/extensions/voronoi.py
--- inkscape_0.48.1-2ubuntu5~/share/extensions/voronoi.py	2011-05-30 17:38:10.000000000 +0100
+++ inkscape_0.48.1-2ubuntu5/share/extensions/voronoi.py	2011-05-30 17:43:57.913386425 +0100
@@ -1,3 +1,4 @@
+#! /usr/bin/env python
 #############################################################################
 #
 # Voronoi diagram calculator/ Delaunay triangulator
